support/cas-server-support-inwebo-mfa/src/main/java/org/apereo/cas/support/inwebo/web/flow/actions/CheckAuthenticationAction.java

```diff
@@ -0,0 +1,72 @@
+package org.apereo.cas.support.inwebo.web.flow.actions;
+
+import org.apereo.cas.support.inwebo.authentication.InweboCredential;
+import org.apereo.cas.support.inwebo.service.InweboService;
+import org.apereo.cas.support.inwebo.service.response.Result;
+import org.apereo.cas.web.flow.resolver.CasWebflowEventResolver;
+import org.apereo.cas.web.support.WebUtils;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import lombok.val;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.context.MessageSource;
+import org.springframework.context.i18n.LocaleContextHolder;
+import org.springframework.webflow.action.AbstractAction;
+import org.springframework.webflow.execution.Event;
+import org.springframework.webflow.execution.RequestContext;
+
+/**
+ * A web action to check the push notification or the OTP generated on the browser.
+ *
+ * @author Jerome LELEU
+ * @since 6.4.0
+ */
+@RequiredArgsConstructor
+@Slf4j
+public class CheckAuthenticationAction extends AbstractAction implements WebflowConstants {
```

Comment:
You shouldn't need to implement a constants interface to use what's in it, unless it actually represents a type. Remove the clause, and use the constants directly with a qualified class name to be easy on the reader.

Reply:
OK. Will do.